src/Processing/ImportProcessingService.php

```diff
@@ -331,7 +331,9 @@ public function generateTransformationResultPreview(array $importDataRow, Mappin
             $data = $operator->generateResultPreview($data);
         }
 
-        if (empty($data)) {
+        if ($data === '0') {
+            return '0';
+        } elseif (empty($data)) {
             return '-- EMPTY --';
         } elseif (is_string($data)) {
             return $data;
```

Comment:
```suggestion
       if (is_string($data) && $data !== '') {
            return $data;
        } elseif (empty($data)) {
            return '-- EMPTY --';
```
@robertSt7 Maybe it would be a good idea to rearrange the statements, add the check for empty string and spare the additonal if condition. Wdyt?